src/examples/liquid_fill_gauge/liquid_fill_gauge.ts

```diff
@@ -222,6 +222,9 @@ const vis: LiquidFillGaugeVisualization = {
       if (gaugeConfig.maxValue <= 0) {
         this.addError({ group: 'config', title: 'Max value must be greater than zero.' })
         return
+      } else if (data.length === 0) {
```

Comment:
Is this PR still a WIP? I don't see how this could fix the issue.
If data has a length property, it would not be falsey unless it is 0, so the original code works.
If data does not exist this would throw an error, which this would not change.

Reply:
![image](https://user-images.githubusercontent.com/56001784/72091300-b3d0b600-32c4-11ea-9bb1-04450d470224.png)

It's my understanding that when No Results are returned from the query to the vis, an empty data array is still passed to the vis. This case tests for that empty array by evaluating if its length is 0.

Reply:
My apologies, my email linked me to an individual commit on this PR where you'd changed this from a `==` to a `===` and somehow I thought that was the only change. Looking at the whole thing this is exactly what I'd expect. 👍